runbot/runbot.py

```diff
@@ -957,6 +958,9 @@ def _cmd(self, cr, uid, ids, context=None):
                     os.mkdir(datadir)
                 cmd += ["--data-dir", datadir]
 
+            if build.branch_id.test_tags:
+                cmd.extend(['--test_tags', "'%s'" % build.branch_id.test_tags])
```

Comment:
no need to double quote argument

Reply:
Hmmm, and what if the tags begins with a '-' like '-standard' ? It could be interpreted as another option/argument.

Reply:
yuck

Reply:
Well, it's the way that one can exclude tests with tags actually. This will be refactored in another task AFIK. Probably with a '-E' option to exclude tags.